### PR TITLE
docs(middleware): fastify wildcard routes

### PR DESCRIPTION
**Pull request by fremgenc**
Adds documentation for issue:

https://github.com/nestjs/nest/issues/972#issuecomment-516707805

## PR Checklist
Please check if your PR fulfills the following requirements:

- [*] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[*] Other... Please describe:
Documentation
```

## What is the current behavior?

Issue Number: [972](https://github.com/nestjs/nest/issues/972)

## Does this PR introduce a breaking change?
```
[ ] Yes
[*] No
```

## Other information

### CODE REVIEW

File: content/middlewares.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ export class AppModule {
 }
 ```
 
-> info **Hint** The `configure()` method can be made asynchronous using `async/await` (e.g., you can `await` completion of an asynchronous operation inside the `configure()` method body).
+> warning **Warning** The `configure()` method can be made asynchronous using `async/await` (e.g., you can `await` completion of an asynchronous operation inside the `configure()` method body).
```
Comment (prateekkathal) [inline content/middlewares.md]: @johnbiundo Don't see any relevance of replace quote type to warning. The statement doesn't even sound like a warning when you read it 😅

File: content/middlewares.md [hunk 2]
```diff
@@ -132,6 +132,8 @@ forRoutes({ path: 'ab*cd', method: RequestMethod.ALL });
 
 The `'ab*cd'` route path will match `abcd`, `ab_cd`, `abecd`, and so on. The characters `?`, `+`, `*`, and `()` may be used in a route path, and are subsets of their regular expression counterparts. The hyphen ( `-`) and the dot (`.`) are interpreted literally by string-based paths.
 
+> info **Hint** Route wildcards do not work with fastify platform applications, since fastify [does not support wildcard routes](https://github.com/nestjs/nest/issues/972#issuecomment-516707805).
```
Comment (prateekkathal) [inline content/middlewares.md]: I feel this should be `> warning **Warning**`

File: content/middlewares.md (hunk 1 above)
Comment (johnbiundo) [reply content/middlewares.md]: Agree.

File: content/middlewares.md (hunk 2 above)
Comment (johnbiundo) [reply content/middlewares.md]: Agree

**Comment by johnbiundo** [comment]
@fremgenc Would you mind making the suggested changes please?

**Comment by fremgenc** [comment]
Suggested changes pushed.

**Comment by kamilmysliwiec** [comment]
Clarified in this PR https://github.com/nestjs/docs.nestjs.com/pull/1053

**Pull request closed without merging**